### PR TITLE
Trm 22818/annotation score visualisation

### PR DESCRIPTION
**Pull request by dlrice**
# Notes

- This needs this Franklin PR https://github.com/ebi-uniprot/franklin-sites/pull/43 to be merged first as there is an error about `div`s being children of `p`s.
- Provided two different IDs to UniProtEvidenceTag to avoid encountering 'two children with the same key' warning during tests.

# Checklist
- [x] What is the link to the Jira that this PR is for?
https://www.ebi.ac.uk/panda/jira/browse/TRM-22818
- [x] Have you written tests and do these have >= 75% coverage of the code that you are contributing?
100%
- [x] Is it possible to visually evaluate this PR in the browser? If so, what are the steps to do this (eg specific URLs or sections)?
In results and protein summary
- [x] Are there any linting errors that required you to add exceptions?
N/A
- [x] If there are backend requests, have you checked for the existence of the resource's attributes before attempting access?
N/A

### CODE REVIEW

File: src/view/uniprotkb/components/ProteinOverviewView.tsx [hunk 1]
```diff
@@ -18,6 +19,7 @@ export const ProteinOverview: FC<{
     _ => _.recommendedName.fullName.value
```
Comment (its-mahdi) [inline src/view/uniprotkb/components/ProteinOverviewView.tsx]: I do realise you are following the `idx` examples/patterns, but two things comes to my mind when I see the `_` here:

1) In most FP languages, `_` is where the data is ignored. Basically a placeholder for something that we don't need to access.

2) A utility function from `Lodash` library.

Does it make sense to use a better parameter name here e.g. `o` or `obj`.

File: src/view/uniprotkb/components/ProteinOverviewView.tsx [hunk 2]
```diff
@@ -42,7 +44,11 @@ export const ProteinOverview: FC<{
     },
     {
       title: 'Annotation score',
-      content: <Bubble value={annotationScore} />,
+      content: (
+        <AnnotationScoreDoughnutChart>
+          {annotationScore}
```
Comment (its-mahdi) [inline src/view/uniprotkb/components/ProteinOverviewView.tsx]: Would it be cleaner and more succinct if `{annotationScore}` was passed as `prop` to `AnnotationScoreDoughnutChart` e.g.

    <AnnotationScoreDoughnutChart score={annotationScore} />

File: src/view/uniprotkb/components/ProteinOverviewView.tsx (hunk 1 above)
Comment (dlrice) [reply src/view/uniprotkb/components/ProteinOverviewView.tsx]: Interesting point, I'll raise this as a broader question with you both (and @xwatkins) as throughout our code base (and in Franklin) we follow this pattern. Would prefer to fix - if that is the consensus - in a separate jira/issue and not as part of this PR.

File: src/view/uniprotkb/components/ProteinOverviewView.tsx (hunk 2 above)
Comment (dlrice) [reply src/view/uniprotkb/components/ProteinOverviewView.tsx]: Good point. Updated.

File: src/view/uniprotkb/components/ProteinOverviewView.tsx (hunk 1 above)
Comment (dlrice) [reply src/view/uniprotkb/components/ProteinOverviewView.tsx]: Updated.

**Pull request merged**